addresses/frontend-mainnet.addresses.json

```diff
@@ -1,7 +1,7 @@
 {
     "addresses": {
         "balancerVaultAddress": "0xBA12222222228d8Ba445958a75a0704d566BF2C8",
-        "convergentPoolFactoryAddress": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD",
+        "convergentPoolFactoryAddress": "0xE88628700eaE9213169D715148ac5A5F47B5dCd9",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏For the UI, we'll need to support both ccpool factories until we sunset the old terms. Can you add this new one as `convergentPoolV2FactoryAddress` so we can keep both?

Reply:
This would lead to a downstream code change which we'd prefer to avoid. Perhaps a deprecated factory field which attaches them to version?

Reply:
Yeah that would work, something like this?

```js
{
  addresses: {
    balancerVaultAddress: "0x.....",
    convergentPoolFactoryAddress: "0xV1.1Address..."
    .....
  },

  v1Addresses: {
    convergentPoolFactoryAddress: "0xOldAddress..."
  }
}

Reply:
to keep it simplest on our end, I think this might work if it jives well with the front end:
```
{
    "addresses": {
        "balancerVaultAddress": "0x...",
        "convergentPoolFactoryAddress": "0xE88...",
        "deprecatedCCPoolFactoryAddress": {
            "v1": "0xOldaddress..."
            }
      }
}
```

Reply:
ya this looks reasonable to me